fetcher/fetcher_test.go

```diff
@@ -0,0 +1,123 @@
+package fetcher
+
+import (
+	"context"
+	"fmt"
+	"github.com/algorand/go-algorand-sdk/client/v2/algod"
+	"github.com/algorand/go-algorand/data/bookkeeping"
+	"github.com/algorand/go-algorand/protocol"
+	"github.com/algorand/go-algorand/rpcs"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
```

Comment:
nit: we group imports by builtin / 3rd party / 1st party. I think the CI would have warned about this if it was working properly on forks.